home-work-app/src/app/redux/reducers/movieEditor.ts

```diff
@@ -3,23 +3,27 @@ import { MovieAction, MovieActionReducer, MovieState } from '../../types';
 const initialState: MovieState = {
     movies: [],
     isUpdated: false,
-    queryParams: {}
+    queryParams: {},
 };
 
-export const movieEditor: MovieActionReducer = (state: MovieState = initialState, action: MovieAction): MovieState => {
+export const movieEditor: MovieActionReducer = (
+    state: MovieState = initialState,
+    action: MovieAction
+): MovieState => {
     switch (action.type.toUpperCase()) {
```

Comment:
There should be much more than just `UPADTE` and `UPDATE_FINISHED` actions.
For all operations in your application you have just 2 actions, it is wrong.
You should have at list actions for sorting, filtering and searching, also `UPDATE` should be renamed to the `SET_MOVIES_LIST`, because `UPDATE` name tells me nothing about what kind of update it is and what exactly will be updated.